gnat2goto/ireps/ireps_generator.py

```diff
@@ -623,7 +623,7 @@ def rec(sn):
 
                 friendly_name = list_schema["friendly_name"]
                 element_type  = list_schema["schema"]
-                register_sub_setter(sn, i, friendly_name, element_type, True)
+                register_sub_setter(sn, i, friendly_name, element_type, True, None)
```

Comment:
Why is `None` added here as an argument, when there is no change to the function's parameters in the definition of the function?

Reply:
Because it was forgotten by the original author. For some reason, python considered this fine but pylint noted this as an error.